docs/advanced_reference/commands/autonomy_analyse.md

```diff
@@ -274,3 +311,51 @@ To aggregate stats for `consensus` block type in the second period, execute:
 ```bash
     autonomy analyse benchmarks abci_build/persistent_data/benchmarks --period 2 --block-type consensus
 ```
+
+## `autonomy analyse service`
+
+Analyse if the service is ready to be deployed or not.
+
+This tool can be used to analyse a service definition and see if there are any potential issues with configuration which can cause issues when running the deployment.
```

Comment:
I think we should explain the current limitations: does this tool get errors at service, agent & skill level? Does it try to build the  service to check if there are errors?

Reply:
I'll ask jose to do a review of the command and update the documentation accordingly. As per your second question, no we currently do not build a service to check if the setting is correct since we already have tools to do that and writing once more tool is not going to be helpful.